From 8f252c90de8baaa1230913bfeb1278214b51405c Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Mon, 20 Aug 2018 07:59:13 -0600
Subject: [PATCH 4/4] manifest: Re-add top-level Android.bp symlink

* The build system really, really, wants this one to stick around

Change-Id: Ie1956138d8c19d321a1806f44d9a3bad8b23fdae
---
 default.xml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/default.xml b/default.xml
index c122bca..d21c97a 100644
--- a/default.xml
+++ b/default.xml
@@ -32,6 +32,7 @@
   <project path="build/blueprint" name="platform/build/blueprint" groups="pdk,tradefed" remote="aosp" />
   <project path="build/kati" name="LineageOS/android_build_kati" groups="pdk,tradefed" />
   <project path="build/soong" name="LineageOS/android_build_soong" groups="pdk,tradefed" >
+    <linkfile src="root.bp" dest="Android.bp" />
     <linkfile src="bootstrap.bash" dest="bootstrap.bash" />
   </project>
   <project path="art" name="LineageOS/android_art" groups="pdk" />
-- 
2.17.1

